migrations/change_sets.go

```diff
@@ -598,3 +598,14 @@ func encodeAccounts(s *changeset.ChangeSet) ([]byte, error) {
 }
 
 // ---- Copy-Paste of code to decode ChangeSets: End -----
+
```

Comment:
Don’t need to clear them - because nobody have data there, also can don’t delete buckets - just leave them, new nodes will not have them if remove bucket from buckets.go